client/blocks/site/style.scss

```diff
@@ -172,9 +172,13 @@
 }
 
 .site__badge {
-	color: var( --sidebar-gridicon-fill );
+	color: var( --color-neutral );
 	padding-right: 4px;
 	line-height: 0;
 	position: relative;
 	vertical-align: middle;
 }
+
+.layout__secondary .site__badge {
```

Comment:
this should really be in layout's styles, not here in the site block. layout is overriding the default, so it should be the container for the style.

Reply:
I thought that too, but that seems to be the common pattern throughout Calypso.

https://github.com/Automattic/wp-calypso/blob/19de94ff427018d1c06bbf926eaeabe6cceec73f/client/blocks/app-promo/style.scss#L45

There's only a couple of them tho, should they all be moved over?

Reply:
Draft PR (still some others to add) to move them over in #31642 :)